### PR TITLE
ci: use `actions/setup-node@v2`

### PR DESCRIPTION
**Pull request by snitin315**
This PR contains a:

- [ ] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [ ] **test update** 
- [ ] **typo fix**
- [X] **metadata update**

### Motivation / Use-Case
ci: use `actions/setup-node@v2`

### CODE REVIEW

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -83,9 +81,6 @@ jobs:
       - name: Install dependencies
         run: npm ci
 
-      - name: Install webpack ${{ matrix.webpack-version }}
-        run: npm i webpack@${{ matrix.webpack-version }}
-
```
Comment (snitin315) [inline .github/workflows/nodejs.yml:88]: This is unnecessary and it's installing `schema-utils: "3.1.0"` which is breaking CI, probably in all loaders/plugins repo where we don't have schema-utils as a direct dependency.

Comment (snitin315) [reply .github/workflows/nodejs.yml:88]: /cc @alexander-akait

Comment (alexander-akait) [reply .github/workflows/nodejs.yml:88]: Just update lock file

Comment (snitin315) [reply .github/workflows/nodejs.yml:88]: No luck here 😕  https://github.com/webpack-contrib/imports-loader/pull/104

**Pull request merged**